main.c

```diff
@@ -67,6 +28,11 @@ void testPairToNumber(
     assert(pairNumber == expectedPairNumber);
 }
 
+void testPrintManual(void)
+{
+    PrintColourCode();
+}
+
```

Comment:
Good separation of test code from production code. Consider a better name for this file